tools/tck-api/src/main/scala/org/opencypher/tools/tck/api/Scenario.scala

```diff
@@ -96,12 +96,22 @@ case class Scenario(categories: List[String], featureName: String, number: Optio
 
           case (ctx, RegisterProcedure(signature, table, _)) =>
             ctx.graph match {
-              case support: ProcedureSupport =>
-                support.registerProcedure(signature, table)
+              case procedureSupport: ProcedureSupport =>
+                Right(procedureSupport.registerProcedure(signature, table))
               case _ =>
             }
             Right(ctx)
```

Comment:
Sorry for the half-edit. I understand the purpose. But `Right(procedureSupport.registerProcedure(signature, table))` is not possible since `registerProcedure()` returns `Unit`. So it will have to be the same way it was to begin with.

Reply:
You are right.